var/spack/repos/builtin/packages/paraview/package.py

```diff
@@ -145,6 +145,40 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
         msg="Use paraview@5.9.0 with %xl_r. Earlier versions are not able to build with xl.",
     )
 
+    # CUDA ARCH
+
+    # This is (more or less) the mapping hard-coded in VTK-m logic
+    # see https://gitlab.kitware.com/vtk/vtk-m/-/blob/v2.1.0/CMake/VTKmDeviceAdapters.cmake?ref_type=tags#L221-247
+    supported_cuda_archs = {
+        "20": "fermi",
+        "21": "fermi",
+        "30": "kepler",
+        "32": "kepler",
+        "35": "kepler",
+        "37": "kepler",
+        "50": "maxwel",
+        "52": "maxwel",
+        "53": "maxwel",
+        "60": "pascal",
+        "61": "pascal",
+        "62": "pascal",
+        "70": "volta",
+        "72": "volta",
+        "75": "turing",
+        "80": "ampere",
+        "86": "ampere",
+    }
+
+    # VTK-m and transitively ParaView does not support Tesla Arch
+    for _arch in range(10, 14):
+        conflicts("cuda_arch=%d" % _arch, when="+cuda", msg="ParaView requires cuda_arch >= 20")
+
+    # Starting from cmake@3.18, CUDA architecture managament can be delegated to CMake.
+    # Hence, it is possible to rely on it instead of relying on custom logic updates from VTK-m for
+    # newer architectures (wrt mapping).
+    for cuda_arch in filter(lambda x: x > 86, map(int, CudaPackage.cuda_arch_values)):
```

Comment:
```suggestion
    for cuda_arch in [arch for arch in CudaPackage.cuda_arch_values if arch > 86]:
```

Reply:
Good point. See b15db552eb2480c1afdc0e116bea00bffbaf4811 where I added also the `int` casting for the comparison.